test/utils.js

```diff
@@ -1,17 +1,23 @@
-'use strict'
+"use strict";
 
-const fs = require('fs')
-const path = require('path')
+const fs = require("fs");
+const path = require("path");
+const { suite } = require("uvu");
 
 const fixture = (dir, file) => {
-  return fs.readFileSync(path.resolve('test', dir, 'fixtures', file), 'utf8')
-}
+  return fs.readFileSync(path.resolve("test", dir, "fixtures", file), "utf8");
+};
 
 const expected = (dir, file) => {
-  return fs.readFileSync(path.resolve('test', dir, 'expect', file), 'utf8')
-}
+  return fs.readFileSync(path.resolve("test", dir, "expect", file), "utf8");
+};
+
+const describe = (name, hook) => {
+  hook(suite(name));
```

Comment:
This file code style if out of Prettier config

Reply:
Fixed with run `pnpm lint`